app/config/security.yml

```diff
@@ -8,12 +8,28 @@ security:
       memory: ~
     admin:
       id: prestashop.security.admin.provider
+    oauth2:
+      memory:
```

Comment:
Are you using in memory user for test purposes?

Reply:
Yep! As we don't have any user interface on the BO yet to manage API clients 😅

Reply:
I think this configuration should be in dev environment, to prevent this code running in production (and if we somehow forget to remove it).